sdk/communication/Azure.Communication.Common/src/CommunicationTokenRefreshOptions.cs

```diff
@@ -39,6 +46,21 @@ public class CommunicationTokenRefreshOptions
             Argument.AssertNotNull(tokenRefresher, nameof(tokenRefresher));
             RefreshProactively = refreshProactively;
             TokenRefresher = tokenRefresher;
+            RefreshOffsetTime = new TimeSpan(0, 0, DefaultExpiringOffsetMinutes, 0);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="CommunicationTokenRefreshOptions"/>.
+        /// </summary>
+        /// <param name="refreshProactively">Indicates whether the token should be proactively renewed prior to expiry or renew on demand.</param>
+        /// <param name="tokenRefresher">The function that provides the token acquired from CommunicationIdentityClient.</param>
+        /// <param name="refreshOffsetTime">Proactive refresh interval.</param>
```

Comment:
I think this needs a better explanation. An interval of N means that something happens every N time.

```suggestion
        /// <param name="refreshOffsetTime">The time span before token expiry that `tokenRefresher` will be called if `refreshProactively` is true. For example, setting it to 5min means that 5min before the cached token expires, proactive refresh will request a new token.</param>
```